packages/modules/mysql/src/mysql-container.test.ts

```diff
@@ -94,9 +94,14 @@ describe("MySqlContainer", () => {
   it("should execute a query and return the result", async () => {
     const container = await new MySqlContainer().start();
 
+    // Test non-root user
     const queryResult = await container.executeQuery("SELECT 1 as res");
     expect(queryResult).toEqual(expect.stringContaining("res\n1\n"));
 
+    // Test root user
+    const rootQueryResult = await container.executeQuery("SELECT CURRENT_USER() as user");
```

Comment:
How does this test test the new `isRoot` parameter?

Reply:
Here we go, fixed. I apologise for the incomplete PR, I don't know what happened.

I kept test root and non-root in the same test as is done in the "should work with database URI" test